packages/nextjs-contentful-renderer/config/jest/setupTests.ts

```diff
@@ -1,5 +0,0 @@
-process.env.CONTENTFUL_SPACE_ID = 'space';
-process.env.CONTENTFUL_ACCESS_TOKEN_PREVIEW = 'preview_token';
-process.env.CONTENTFUL_ACCESS_TOKEN_DELIVERY = 'delivery_token';
-process.env.SITE_DOMAIN = 'site_domain';
-process.env.CONTENTFUL_ENVIRONMENT = 'master';
```

Comment:
don't we need this env vars anymore?

Reply:
Yeah. I think we do need them. I will pull them back.

Reply:
Done. File reverted.